test/framework/src/main/java/org/elasticsearch/indices/analysis/AnalysisFactoryTestCase.java

```diff
@@ -129,25 +129,23 @@ private static String toCamelCase(String s) {
 
     static final Map<String,Class<?>> KNOWN_TOKENIZERS = new MapBuilder<String,Class<?>>()
         // exposed in ES
-        .put("classic",       ClassicTokenizerFactory.class)
-        .put("edgengram",     EdgeNGramTokenizerFactory.class)
-        .put("keyword",       KeywordTokenizerFactory.class)
-        .put("letter",        LetterTokenizerFactory.class)
-        .put("lowercase",     LowerCaseTokenizerFactory.class)
-        .put("ngram",         NGramTokenizerFactory.class)
-        .put("pathhierarchy", PathHierarchyTokenizerFactory.class)
-        .put("pattern",       PatternTokenizerFactory.class)
-        .put("standard",      StandardTokenizerFactory.class)
-        .put("thai",          ThaiTokenizerFactory.class)
-        .put("uax29urlemail", UAX29URLEmailTokenizerFactory.class)
-        .put("whitespace",    WhitespaceTokenizerFactory.class)
+        .put("classic",            ClassicTokenizerFactory.class)
+        .put("edgengram",          EdgeNGramTokenizerFactory.class)
+        .put("keyword",            KeywordTokenizerFactory.class)
+        .put("letter",             LetterTokenizerFactory.class)
+        .put("lowercase",          LowerCaseTokenizerFactory.class)
+        .put("ngram",              NGramTokenizerFactory.class)
+        .put("pathhierarchy",      PathHierarchyTokenizerFactory.class)
+        .put("pattern",            PatternTokenizerFactory.class)
+        .put("simplepattern",      MovedToAnalysisCommon.class)
+        .put("simplepatternsplit", MovedToAnalysisCommon.class)
+        .put("standard",           StandardTokenizerFactory.class)
+        .put("thai",               ThaiTokenizerFactory.class)
+        .put("uax29urlemail",      UAX29URLEmailTokenizerFactory.class)
+        .put("whitespace",         WhitespaceTokenizerFactory.class)
```

Comment:
This formatting is annoying, since it has to be maintained every time a tokenizer is added removed. I appreciate you trying to maintain it but maybe we should take this opportunity to just format it normally so we do not have to worry about that?